### ISSUE TITLE
sometimes it treats the mouse as if it were one line below where it actually is

### ISSUE THREAD

**Issue by AlexKnauth** [miraleung/racketeer#2]
I was playing around with this a bit and I noticed something weird and I’m not sure what happened.
When I clicked the cursor somewhere, DrRacket behaved as if I had clicked one line below where I actually clicked.  The check-syntax arrows also behaved as if my mouse was one line below where it actually was.

When this happened, there was a light grey box above the #lang line, and if I click in that grey box it behaves as if I clicked in the #lang line. It seemed like DrRacket was displaying the grey box but pretending it’s not there.  
Then later I opened a new tab and the box went away, and it was back to normal again.

**Comment by miraleung** [comment]
What actions did you take to make the grey box appear? The check-syntax arrows aren't part of Racketeer, so I'd like to investigate if this is a DrRacket or Racketeer-specific issue.

Thanks for bringing this up.

**Comment by AlexKnauth** [comment]
I have no idea what I did to make the grey box appear.  I was just playing around with writing tests for, and then writing, a fibonacci function, sort of like in the example in the README (except I was doing it in #lang sweet-exp racket, but that shouldn't matter), and one time I just clicked on one of the tests and the cursor appeared a line below where I clicked. I didn't notice the grey box until later.

As for the check-syntax arrows, that's probably just because something fooled DrRacket into thinking that the things in the Definitions window were a line down (or up, depending on how you think about it) from where they actually were. I just mentioned the check-syntax arrows to say that it wasn't just a problem with clicking, it was a problem with everything having to do with the mouse.
